### ISSUE TITLE
Suggestion: Option to recover hit dice before spending them

### ISSUE THREAD

**Issue by GambetTV** [schultzcole/FVTT-Long-Rest-HD-Healing-5e#8]
Right now when you select Long Rest, you are given the option to spend Hit Dice, but only before the Long Rest goes into effect. So if you go into the Long Rest with 0 Hit Dice remaining, you cannot heal during that long rest. The rules are ambiguous about what order this should apply, but it would be great if this module provided the option to DMs to do it in either order.

**Comment by misthero** [comment]
I totally agree with this request, but for different reasons. our rules are: you rest and as effect of rest you recover dices, only after you recovered the dices you can use it.

what we do right now is restig without using hit dices an immediately open the rest dialog again and use the dices, this time without hitting the rest button but simply close the dialog

**Comment by schultzcole** [comment]
Hi folks, sorry for not responding to this earlier. I'll look in to adding this after the next dnd5e update (1.3.0)

**Comment by misthero** [comment]
thankyou, looking forward for this, keep up the good work!
